src/convertToObject.js

```diff
@@ -6,7 +6,26 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString
+    .trim()
+    .split(';')
+    .filter((style) => style.trim() !== '');
```

Comment:
The chaining of methods here is well done, but you should be aware that the filter condition will not remove empty style declarations that contain only whitespace. Instead of checking for an empty string after trimming, it would be more robust to check if the trimmed string is not empty.